docs/excel/excel-add-ins-conditional-formatting.md

```diff
@@ -0,0 +1,305 @@
+
+Only one format type can be set for the ConditionalFormat object. This is determined by the `type` property, which is a [ConditionalFormatType](https://docs.microsoft.com/javascript/api/excel/excel.conditionalformattype) enum. `type` us set when adding a conditional format to a range. 
+
+## Creating conditional formatting rules
+
+Conditional formats are added to a range by using `conditionalFormats.add`. Once added, the properties specific to the conditional format can be set. The following examples show the creation of different formatting types.
+
+### [Cell value](https://docs.microsoft.com/javascript/api/excel/excel.cellvalueconditionalformat)
+
+Cell value conditional formatting applies a user-defined format based on the results of one or two formulas in the [ConditionalCellValueRule]( https://docs.microsoft.com/javascript/api/excel/excel.conditionalcellvaluerule). The `operator` property is a [ConditionalCellValueOperator]( https://docs.microsoft.com/javascript/api/excel/excel.conditionalcellvalueoperator) defining how the resulting expressions relate to the formatting. The following example shows red font coloring applied to any value in the range less than zero.
+
+![A range with negative numbers in red.](../images/excel-conditional-format-cell-value.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B21:E23");
+const conditionalFormat = range.conditionalFormats.add(
+    Excel.ConditionalFormatType.cellValue
+);
+
+// set the font of negative numbers to red
+conditionalFormat.cellValue.format.font.color = "red";
+conditionalFormat.cellValue.rule = { formula1: "=0", operator: "LessThan" };
+
+await context.sync();
+```
+
+### [Color scale](https://docs.microsoft.com/javascript/api/excel/excel.colorscaleconditionalformat)
+
+Color scale formatting applies a color gradient across the data range. The `criteria` property on the `ColorScaleConditionalFormat` defines the colors for the minimum, maximum, and an optional midpoint. Every cell will be proportionally colored based on its value relative to the minimum, midpoint, and maximum. The following example shows a range being colored blue to yellow to red.
+
+![A range with the low number in blue, average number in yellow, and high number is red, with gradients for between values.](../images/excel-conditional-format-color-scale.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B2:M5");
+const conditionalFormat = range.conditionalFormats.add(
+      Excel.ConditionalFormatType.colorScale
+);
+
+// color the backgrounds of the cells from blue to yellow to red based on value
+const criteria = {
+      minimum: {
+           formula: null,
```

Comment:
Need an explanation for why the minimum and maximum formulas are `null`.

Reply:
Sent an offline follow up question about this.

Reply:
@Rick-Kirkham Added an expanded description of the formula and type properties.